indico/modules/events/editing/client/js/components/ConditionInfo.jsx

```diff
@@ -0,0 +1,103 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2020 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import editReviewConditionURL from 'indico-url:event_editing.api_edit_review_condition';
+
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import {useSelector} from 'react-redux';
+import {Icon, Label} from 'semantic-ui-react';
+
+import {RequestConfirm, TooltipIfTruncated} from 'indico/react/components';
+import {Translate} from 'indico/react/i18n';
+import {handleSubmitError} from 'indico/react/forms';
+import {handleAxiosError, indicoAxios} from 'indico/utils/axios';
+
+import ReviewConditionForm from './ReviewConditionForm';
+
+import './ConditionInfo.module.scss';
+
+export default function ConditionInfo({condition, index, onUpdate}) {
+  const [isEditing, setIsEditing] = useState(false);
+  const [isDeleting, setIsDeleting] = useState(false);
+  const {eventId} = useSelector(state => state.staticData);
+  const url = editReviewConditionURL({confId: eventId, index});
+
+  const deleteCondition = async () => {
+    try {
+      await indicoAxios.delete(url);
+      if (onUpdate) {
+        onUpdate();
+      }
```

Comment:
I don't think this should be in the try block - unless it can throw something as well.